packages/modules/redis/src/redis-container.test.ts

```diff
@@ -119,4 +119,84 @@ describe("RedisContainer", { timeout: 240_000 }, () => {
     client.destroy();
     // }
   });
+
+  it("should start redis with custom command", async () => {
```

Comment:
The redis containers aren't stopped at the end of the tests. Let's see if we can follow the existing `await using` syntax to have this handled automatically for us

Reply:
Here I couldn’t use the using because I need to separate the start from the container to access the createOpts only available in the GenericContainer class, once started we are on a StartedContainer instance.

If you have a solution to access the createOpts on a StartedContainer I am interested